FSharp-4.1/FS-1004-result-type.md

```diff
@@ -16,8 +16,8 @@ In order to be able to write code that you can easily consume that does not thro
 it's useful to have a fsharp type who contains the result on success or the error on failure
 
 ```fsharp
-type Result<'TSuccess,'TError> = 
-     | Success of 'TSuccess 
+type Result<'TOk,'TError> = 
+     | Ok of 'TSuccess 
```

Comment:
It have to be `Ok of 'TOk`

Reply:
Since this is already commited, did I create a new PR: https://github.com/fsharp/fslang-design/pull/728/files

Thanks for spotting it!